config.json

```
@@ -1949,6 +1949,15 @@
           "pattern_matching",
           "strings"
         ]
+      },
+      {
+        "slug": "state-of-tic-tac-toe",
+        "name": "State of Tic Tac Toe",
+        "uuid": "ab6050ba-b40c-46bb-938f-7baef097b52d",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1,
```

```suggestion
        "difficulty": 5,
```

I think this exercise is a 5, which classifies it as a "medium" difficulty exercise. The julia track which originally introduced this exercise also has it as a 5, so that sounds about right.